lib/13/libLLVM_h.jl

```diff
@@ -1956,6 +1956,8 @@ end
     LLVMMSP430BUILTINCallConv = 94
     LLVMAMDGPULSCallConv = 95
     LLVMAMDGPUESCallConv = 96
+    LLVMMETALFUNCCallConv = 102
```

Comment:
Do we have 97-101?

Reply:
[LLVM 15 does](https://llvm.org/doxygen/namespacellvm_1_1CallingConv.html)